src/sync/sync_manager.hpp

```diff
@@ -104,7 +106,7 @@ friend class SyncSession;
     std::shared_ptr<SyncUser> get_user(const std::string& identity,
                                        std::string refresh_token,
                                        util::Optional<std::string> auth_server_url=none,
-                                       bool is_admin=false);
+                                       bool should_persist=true);
```

Comment:
This should be renamed `is_ephemeral` and default to `false`, since if we reverse the polarity of this bool silently we'll just end up introducing bugs into bindings.